### PR TITLE
API Pipeline secrets 🕵️‍♂️

### PR DESCRIPTION
**Pull request by pataruco**
# API Pipeline secrets 🕵️‍♂️

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/recbVwgdO7MEOYziQ?blocks=hide)
GitHub Issue #376 

### Acceptance Criteria

- [ ] API POD to consume environment variables kubernetes secret

### Testing information

1. Go to `infrastructure/environments/development`folder
```sh
infrastructure/environments/development
```
2. Source environmental variables
```sh
source .env
```
3. Refresh cluster credentials
```sh
../../scripts/create-kubernetes-config.sh
```
4. Select option **`3) dev`**
5. See **`azure-search-secrets`** in the cluster
```sh
kubectl describe secrets/azure-search-secrets
```
6. 🍾

### CODE REVIEW

File: medicines/api/README.md [hunk 1]
```diff
@@ -43,25 +43,38 @@ To see the graphql explorer, go to http://127.0.0.1:8080/graphiql.
 ## Deploy API pod in Kuberbetes cluster ⎈
 
 1. Navigate to this directory, `/medicines/api`
-2. Create [Kubernetes Service][kubernetes service]
+2. Source environmental variables from the correspondent environment
```
Comment (rjkerrison) [inline medicines/api/README.md]: _"corresponding"_ not _"correspondent"_

File: medicines/api/README.md [hunk 2]
```diff
@@ -43,25 +43,38 @@ To see the graphql explorer, go to http://127.0.0.1:8080/graphiql.
...
+   ```sh
+   source ./infrastructure/non-prod/.env
+   ```
+3. Create [Kubernetes secret][kubernetes secret] for API Pod to consume
```
Comment (rjkerrison) [inline medicines/api/README.md]: will this step need to be repeated each time we deploy an API pod into the cluster?

File: medicines/api/infrastructure/production/.env.example [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export AZURE_SEARCH_INDEX=
```
Comment (rjkerrison) [inline medicines/api/infrastructure/production/.env.example]: `.env` inconsistency here: `export` is not included in the other examples

File: medicines/api/README.md (hunk 2 above)
Comment (pataruco) [reply medicines/api/README.md]: Nope

File: medicines/api/infrastructure/production/.env.example (hunk 3 above)
Comment (pataruco) [reply medicines/api/infrastructure/production/.env.example]: Good catch

File: medicines/api/README.md (hunk 1 above)
Comment (pataruco) [reply medicines/api/README.md]: Thanks

File: medicines/api/README.md (hunk 2 above)
Comment (rjkerrison) [reply medicines/api/README.md]: What does someone do if they want to check whether this step needs to be done?

Comment (pataruco) [reply medicines/api/README.md]: Run the command and if they are created it would notify that

**Pull request merged**